x-pack/legacy/plugins/canvas/server/plugin.ts

```diff
@@ -13,11 +13,10 @@ export class Plugin {
   public setup(core: CoreSetup, plugins: PluginsSetup) {
     routes(core);
 
-    const { serverFunctions } = plugins.interpreter.register({ serverFunctions: functions });
+    plugins.interpreter.register({ serverFunctions: functions });
 
     core.injectUiAppVars('canvas', async () => {
       const config = core.getServerConfig();
-      const basePath = config.get('server.basePath');
       const reportingBrowserType = (() => {
```

Comment:
No need to worry about this, per https://github.com/elastic/kibana/pull/54050/files#r370402125

Leaving this as-is would complicate the Reporting team from upgrading to the NP config service anyway 😄